build/antrea/ovs/Dockerfile.ubi

```diff
@@ -0,0 +1,50 @@
+FROM registry.access.redhat.com/ubi8 as base-rpms
+
+COPY CentOS.repo /tmp/CentOS.repo
+RUN rm -f /etc/yum.repos.d/* && mv /tmp/CentOS.repo /etc/yum.repos.d/CentOS.repo && \
+    curl https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official -o /etc/pki/rpm-gpg/RPM-GPG-KEY-centosofficial && \
+    subscription-manager config --rhsm.manage_repos=0 && \ 
+    yum clean all -y && yum update -y
+
+# Download dependencies for ovs.
+RUN yum install -y epel-release && \ 
+    yum install -y --downloadonly --downloaddir="/tmp/build-packages" unbound python3-sphinx desktop-file-utils groff graphviz selinux-policy-devel && \
+    yum install -y --downloadonly --downloaddir="/tmp/install-packages" strongswan libreswan
+RUN yum install -y --downloadonly --downloaddir="/tmp/install-packages" unbound
```

Comment:
Download missed packages for building and installing OVS from Centos repo, package 'unbound' is required in both building and installing, so download it twice.

Reply:
Can you put this comment in the Dockerfile, it is confused to me we need to download unbound twice

Reply:
> Can you put this comment in the Dockerfile, it is confused to me we need to download unbound twice

yes, i'ill put the comment in Dockerfile. In line 20(https://github.com/vmware/antrea-operator-for-kubernetes/pull/33/files#diff-8ce328e4696f391c24f9cc83502b71c50e9009ec66f361d21f726f213f51f6feR20), the /tmp/build-packages/unbound is copied to next layer to build ovs rpms. And in line 42(https://github.com/vmware/antrea-operator-for-kubernetes/pull/33/files#diff-8ce328e4696f391c24f9cc83502b71c50e9009ec66f361d21f726f213f51f6feR42), the /tmp/install-packages/unbound is copied to install ovs rpms. So the package unbound is downloaded the directory '/tmp/build-packages' and '/tmp/install-packages'.